docs/admin-guide/run-plone.md

```diff
@@ -101,3 +101,11 @@ pip:
     ```
 
 For any of these commands, press {kbd}`ctrl-d` to stop the process.
+
+### Set a "fake" request
+
+To make sure that the request is fully set up for any code that uses `zope.globalrequest.getRequest` you might need to do:
+:   ```python
+    from zope.globalrequest import setRequest
+    setRequest(app.REQUEST)
+    ```
```

Comment:
Please dedent all the above lines because this is not a definition list, but a simple code example.